util/start_test

```diff
@@ -676,7 +676,11 @@ def set_up_environment():
         args.performance = True
         args.gen_graphs = True
         args.compopts += " --fast"
-        if not tgt_platform == "darwin":
+        # Force static linking unless the target requires dynamic.
```

Comment:
Long term, I agree that we will want libnuma other than on Crays.  Elliot was nervous about doing that, so I limited myself to Crays for this PR.